apps/lab/src/app/home.module.css

```diff
@@ -5,4 +5,23 @@
 .hero {
   text-align: center;
   padding-top: 200px;
+}
+
+.btn {
```

Comment:
For now this is ok, because we dont have style support, although these styles should be part of your package as that is where the button is created.